cmd/cstor-pool-mgmt/controller/pool-controller/after_queue_handler.go

```diff
@@ -0,0 +1,192 @@
+package poolcontroller
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+
+	"github.com/golang/glog"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/controller/common"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/pool"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	"github.com/openebs/maya/pkg/util"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/tools/cache"
+)
+
+// syncHandler compares the actual state with the desired, and attempts to
+// converge the two. It then updates the Status block of the cStorPoolUpdated resource
+// with the current status of the resource.
+func (c *CStorPoolController) syncHandler(key, operation string) error {
+	glog.Infof("at sync handler")
+	cStorPoolGot, err := c.getPoolResource(key)
+	if err != nil {
+		return err
+	}
+	status, err := c.cStorPoolEventHandler(operation, cStorPoolGot)
+	if status == common.StatusIgnore {
+		return nil
+	}
+	cStorPoolGot.Status.Phase = status
+	if err != nil {
+		_, err := c.clientset.OpenebsV1alpha1().CStorPools().Update(cStorPoolGot)
+		if err != nil {
+			return err
+		}
+		return err
+	}
+	_, err = c.clientset.OpenebsV1alpha1().CStorPools().Update(cStorPoolGot)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// cStorPoolEventHandler is to handle cstor pool related events.
+func (c *CStorPoolController) cStorPoolEventHandler(operation string, cStorPoolGot *apis.CStorPool) (string, error) {
+	pool.RunnerVar = util.RealRunner{}
+	switch operation {
+	case "add":
+		glog.Info("added event")
```

Comment:
Add more description to the log message - which pool, type of operation.

Reply:
will change